signac/common/config.py

```diff
@@ -19,10 +21,26 @@
 
 
 class PermissionsError(ConfigError):
+    """Indicates an error in file permissions."""
     pass
 
 
+def _search_local(root):
+    for fn in CONFIG_FILENAMES:
+        fn_ = os.path.abspath(os.path.join(root, fn))
+        if os.path.isfile(fn_):
+            yield fn_
+
+
 def search_tree(root=None):
+    """Locates signac configuration files in a directory hierarchy.
+
+    Parameters
+    ----------
+    root : str
```

Comment:
```suggestion
    root : str, optional
```
If it is not necessary to specify a keyword argument, we should use optional.
I read this in the block Sections/Parameters on this [website](https://numpydoc.readthedocs.io/en/latest/format.html_)

Reply:
Okay I just noticed that there's an issue(#344)  open regarding it.